<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>580</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QLabel" name="Capuchino">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>240</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Rockwell Extra Bold</family>
        <pointsize>18</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Capucchino</string>
      </property>
     </widget>
     <widget class="QLabel" name="Latte">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>290</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Rockwell Extra Bold</family>
        <pointsize>18</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Latte</string>
      </property>
     </widget>
     <widget class="QLabel" name="IcedTeas">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>390</y>
        <width>191</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Rockwell Extra Bold</family>
        <pointsize>18</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Iced   Tea</string>
      </property>
     </widget>
     <widget class="QLabel" name="Espresso">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>190</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Rockwell Extra Bold</family>
        <pointsize>18</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Espresso</string>
      </property>
     </widget>
     <widget class="QLabel" name="Frappuccino">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>340</y>
        <width>191</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Rockwell Extra Bold</family>
        <pointsize>18</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Frappuccino</string>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonAddEspresso">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>190</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Add</string>
      </property>
     </widget>
     <widget class="QLabel" name="PriceEspresso">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>190</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Rockwell Extra Bold</family>
        <pointsize>18</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>3.90$</string>
      </property>
     </widget>
     <widget class="QLabel" name="PriceCapuchino">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>240</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Rockwell Extra Bold</family>
        <pointsize>18</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>5.90$</string>
      </property>
     </widget>
     <widget class="QLabel" name="PriceLatte">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>290</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Rockwell Extra Bold</family>
        <pointsize>18</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>5.90$</string>
      </property>
     </widget>
     <widget class="QLabel" name="PriceFrappucchino">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>340</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Rockwell Extra Bold</family>
        <pointsize>18</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>7.90$</string>
      </property>
     </widget>
     <widget class="QLabel" name="PriceIcedTeas">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>390</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Rockwell Extra Bold</family>
        <pointsize>18</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>4.90$</string>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonAddCapucchino">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>240</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Add</string>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonAddLatte">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>290</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Add</string>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonAddFrappucchino">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>340</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Add</string>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonAddIcedTeas">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>390</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Add</string>
      </property>
     </widget>
     <widget class="QLabel" name="logo">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>10</y>
        <width>357</width>
        <height>159</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Rockwell</family>
        <pointsize>28</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="textFormat">
       <enum>Qt::AutoText</enum>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="background">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>800</width>
        <height>600</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Rockwell</family>
        <pointsize>28</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="textFormat">
       <enum>Qt::AutoText</enum>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonViewCart">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>460</y>
        <width>201</width>
        <height>101</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>View Cart</string>
      </property>
     </widget>
     <zorder>background</zorder>
     <zorder>Capuchino</zorder>
     <zorder>Latte</zorder>
     <zorder>IcedTeas</zorder>
     <zorder>Espresso</zorder>
     <zorder>Frappuccino</zorder>
     <zorder>buttonAddEspresso</zorder>
     <zorder>PriceEspresso</zorder>
     <zorder>PriceCapuchino</zorder>
     <zorder>PriceLatte</zorder>
     <zorder>PriceFrappucchino</zorder>
     <zorder>PriceIcedTeas</zorder>
     <zorder>buttonAddCapucchino</zorder>
     <zorder>buttonAddLatte</zorder>
     <zorder>buttonAddFrappucchino</zorder>
     <zorder>buttonAddIcedTeas</zorder>
     <zorder>logo</zorder>
     <zorder>buttonViewCart</zorder>
    </widget>
    <widget class="QWidget" name="page2">
     <widget class="QLabel" name="YourOrder">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>200</y>
        <width>240</width>
        <height>50</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Rockwell</family>
        <pointsize>28</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Your Order</string>
      </property>
      <property name="textFormat">
       <enum>Qt::AutoText</enum>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QListWidget" name="itemsInCart">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>270</y>
        <width>400</width>
        <height>141</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="Total">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>430</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Rockwell Extra Bold</family>
        <pointsize>18</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Total:</string>
      </property>
     </widget>
     <widget class="QLabel" name="TotalPriceInCart">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>430</y>
        <width>291</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Rockwell Extra Bold</family>
        <pointsize>18</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonBackToSelection">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>490</y>
        <width>151</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonPay">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>490</y>
        <width>151</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Pay</string>
      </property>
     </widget>
     <widget class="QLabel" name="background2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>800</width>
        <height>600</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Rockwell</family>
        <pointsize>28</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="textFormat">
       <enum>Qt::AutoText</enum>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="logo2">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>10</y>
        <width>357</width>
        <height>159</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Rockwell</family>
        <pointsize>28</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="textFormat">
       <enum>Qt::AutoText</enum>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <zorder>background2</zorder>
     <zorder>YourOrder</zorder>
     <zorder>itemsInCart</zorder>
     <zorder>Total</zorder>
     <zorder>TotalPriceInCart</zorder>
     <zorder>buttonBackToSelection</zorder>
     <zorder>buttonPay</zorder>
     <zorder>logo2</zorder>
    </widget>
    <widget class="QWidget" name="page3">
     <widget class="QLabel" name="Payment">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>200</y>
        <width>240</width>
        <height>50</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Rockwell</family>
        <pointsize>28</pointsize>
        <italic>false</italic>
        <bold>true</bold>
        <underline>false</underline>
        <strikeout>false</strikeout>
       </font>
      </property>
      <property name="text">
       <string>Payment</string>
      </property>
      <property name="textFormat">
       <enum>Qt::AutoText</enum>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="Total_2">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>280</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Rockwell Extra Bold</family>
        <pointsize>18</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Total:</string>
      </property>
     </widget>
     <widget class="QLabel" name="TotalPriceInCart_2">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>280</y>
        <width>221</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Rockwell Extra Bold</family>
        <pointsize>18</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="Name">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>330</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Rockwell Extra Bold</family>
        <pointsize>18</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Name:</string>
      </property>
     </widget>
     <widget class="QLabel" name="CardNumber">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>380</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Rockwell Extra Bold</family>
        <pointsize>18</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Card Number:</string>
      </property>
     </widget>
     <widget class="QLabel" name="background3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>800</width>
        <height>600</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Rockwell</family>
        <pointsize>28</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="textFormat">
       <enum>Qt::AutoText</enum>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonConfirm">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>460</y>
        <width>201</width>
        <height>60</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Confirm</string>
      </property>
     </widget>
     <widget class="QLabel" name="logo3">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>10</y>
        <width>357</width>
        <height>159</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Rockwell</family>
        <pointsize>28</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="textFormat">
       <enum>Qt::AutoText</enum>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEditName">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>330</y>
        <width>221</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Rockwell</family>
       </font>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEditCardNumber">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>380</y>
        <width>221</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Rockwell</family>
       </font>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonBackToCart">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>460</y>
        <width>201</width>
        <height>60</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
     <zorder>background3</zorder>
     <zorder>Payment</zorder>
     <zorder>Total_2</zorder>
     <zorder>TotalPriceInCart_2</zorder>
     <zorder>Name</zorder>
     <zorder>CardNumber</zorder>
     <zorder>buttonConfirm</zorder>
     <zorder>logo3</zorder>
     <zorder>textEditName</zorder>
     <zorder>textEditCardNumber</zorder>
     <zorder>buttonBackToCart</zorder>
    </widget>
    <widget class="QWidget" name="page4">
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>250</y>
        <width>360</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>28</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Thank you</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>340</y>
        <width>360</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>28</pointsize>
       </font>
      </property>
      <property name="text">
       <string>F R I E N D S</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="background4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>800</width>
        <height>600</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Rockwell</family>
        <pointsize>28</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="textFormat">
       <enum>Qt::AutoText</enum>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="logo4">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>10</y>
        <width>357</width>
        <height>159</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Rockwell</family>
        <pointsize>28</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="textFormat">
       <enum>Qt::AutoText</enum>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonConfirm_2">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>460</y>
        <width>401</width>
        <height>60</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Make a new order</string>
      </property>
     </widget>
     <zorder>background4</zorder>
     <zorder>label</zorder>
     <zorder>label_2</zorder>
     <zorder>logo4</zorder>
     <zorder>buttonConfirm_2</zorder>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
